archive/conf/routes

```diff
@@ -7,5 +7,7 @@ GET        /_healthcheck                            controllers.HealthCheck.heal
 
 GET        /dev/ui                                  controllers.DevComponentController.renderComponent()
```

Comment:
What is dev/ui being used for right now? Is it worth removing (maybe in another PR)?

Reply:
I believe the intention was to make individual components renderable by hitting this endpoint. I may be wrong about this (is that correct, @TBonnin?). It is not currently being used anywhere, so we could delete it, along with [`TestComponent.scala`](https://github.com/guardian/frontend/blob/master/common/app/rendering/TestComponent.scala)

Happy to do this in another PR, I don't want to make too many different changes in this one 😅

Reply:
Potentially it is more sensible to rewrite TestComponent to use `dev/props` as its a valuable test

Reply:
@SiAdcock is right. The goal was to provide an endpoint that can render any component in isolation.
The work @SiAdcock superseeds this which mean the endpoint can be removed.